gamedata/scripts/magazines.script

```diff
@@ -67,7 +67,7 @@ function get_mag_time(mag_section,is_unload)
 	local default_delay=SYS_GetParam(2, "tch_mag_base", "load_delay") or 0.4
 
 	if magazines_mcm.get_config('ignore_loaddelay') then
-		default_delay=1
```

Comment:
This was intentional. the description for the check box says that checking the box turns the slider into a seconds of delay value. 
Other wise it is hard to set to specific value.